File: 백준/Bronze/1978. 소수 찾기/소수 찾기.swift
_ = readLine()

var isPrime = [Bool](repeating: true, count: 1001)
isPrime[1] = false

for i in 2...1000 {
    if !isPrime[i] { continue }
    for j in stride(from: 2 * i, through: 1000, by: i) {
        isPrime[j] = false
    }
}

let arr = readLine()!.split(separator: " ").compactMap { Int($0) }

print(arr.filter { isPrime[$0] }.count)

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
input()
isPrime = [True for i in range(1001)]
isPrime[1] = False

for i in range(2, 1001):
    if not isPrime[i]:
        continue
    for j in range(2 * i, 1001, i):
        isPrime[j] = False

arr = list(map(int, input().split()))

count = 0

for i in arr:
    if isPrime[i]:
        count += 1

print(count)